tests/unit_test_pysd.py

```diff
@@ -184,6 +184,7 @@ def test_func():
         res = model.run(return_columns=['Room Temperature'])
         self.assertEqual(test_func(), res['Room Temperature'].iloc[0])
 
+    @unittest.skip('to be added after fixing issue with string handling in py 3')
```

Comment:
Is this something you are hoping to do in this PR?

Reply:
I hope to do it eventually :) Like before school starts, but not for this pr. Edited the comment and added to the pr.